packages/@aws-cdk/aws-ec2/lib/vpc-flow-logs.ts

```diff
@@ -401,6 +401,115 @@ export enum FlowLogMaxAggregationInterval {
 
 }
 
+/**
+ * The following table describes all of the available fields for a flow log record.
+ */
+export enum LogFormatField {
```

Comment:
> I don't think you need custom for LogFormatField. "custom" in the document below does not mean that the user can specify any Key. It means that you can specify any Key of "Available fields" with a space-separated string.
docs.aws.amazon.com/vpc/latest/userguide/flow-logs.html#flow-logs-fields

The reason we want to have the `custom` option is to allow the user to specify new fields that we
have not added support for yet. We don't want to block users from using new values as soon as they
are released. If you don't like the `custom` methed we can do without that and the
user can just do something like:

```ts
customLogFormatFields: [
  LogFormatField.SRC_PORT,
  new LogFormatField('${new-field}'),
]
```

Reply:
Fixed. Added Usage to Readme.